cdap-api-spark/src/main/java/co/cask/cdap/api/spark/JavaSparkExecutionContext.java

```diff
@@ -445,6 +445,18 @@
   }
 
   /**
+   * Saves the given {@link JavaPairRDD} to the given {@link Dataset}.
+   *
+   * @param rdd the {@link JavaPairRDD} to be saved
+   * @param namespace the namespace for the dataset
+   * @param datasetName name of the Dataset
```

Comment:
In this file, I think you can remove capitalization of 'D'ataset when its not an ``{@link }`` element